soccer/planning/CollectPathPlanner.cpp

```diff
@@ -0,0 +1,432 @@
+#include "CollectPathPlanner.hpp"
```

Comment:
Clean up comments and take out print statements

Reply:
Otherwise this has all been reviewed in #1180 and I think it LGTM.